GitUI/CommandsDialogs/RepoHosting/ForkAndCloneForm.cs

```diff
@@ -438,11 +438,11 @@ private void UpdateCloneInfo(bool updateCreateDirTB = true, bool updateProtocols
 
             if (repo != null)
             {
-                bool multipleProtocols = repo.SupportedCloneProtocols.Length > 0;
+                bool multipleProtocols = repo.SupportedCloneProtocols.Any();
 
                 if (multipleProtocols && updateProtocols)
                 {
-                    var currentSelection = (string)ProtocolDropdownList.SelectedItem;
+                    var currentSelection = (GitProtocol)(ProtocolDropdownList.SelectedItem ?? repo.SupportedCloneProtocols.First());
```

Comment:
This doesn't look safe?
Are we certain this won't fail?

Reply:
It seems to me. We checked that `repo.SupportedCloneProtocols` is not empty 2 lines above.